### PR TITLE
Add GCP as platform for Managed Services

### PR DESCRIPTION
**Pull request by morenod**
There is no defined name for Managed Services on GCP so I have choosed ROGCP. We will have to change it to the selected one when defined

### CODE REVIEW

File: dags/openshift_nightlies/tasks/install/openshift.py [hunk 1]
```diff
@@ -52,7 +53,6 @@ def __init__(self, dag, config: DagConfig, release: OpenshiftRelease):
             **self.azure_creds,
             **self.openstack_creds,
             **self.rosa_creds,
-            **self.rhacs_creds,
```
Comment (Harshith-umesh) [inline dags/openshift_nightlies/tasks/install/openshift.py:55]: why delete this line?

Comment (morenod) [reply dags/openshift_nightlies/tasks/install/openshift.py]: fixed, I dont know why I removed it, maybe a typo during adding the gcp creds. cood catch!

**Review by Harshith-umesh — APPROVED** [review]
LGTM

**Review by whitleykeith — APPROVED** [review]
Lgtm, @morenod will let you merge when you're ready

**Comment by morenod** [comment]
> Lgtm, @morenod will let you merge when you're ready

Merge please, I dont have rights to do it

**Pull request merged**